src/StoreApi/Routes/AbstractRoute.php

```diff
@@ -2,7 +2,7 @@
 namespace Automattic\WooCommerce\Blocks\StoreApi\Routes;
 
 use Automattic\WooCommerce\Blocks\StoreApi\Schemas\AbstractSchema;
-use Automattic\WooCommerce\Blocks\StoreApi\Utilities\InvalidStockLevelsInCartException;
+use Automattic\WooCommerce\Blocks\StoreApi\Utilities\InvalidCartException;
```

Comment:
This will need an updated namespace now [v1 has merged](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5911).